### PR TITLE
docs: Document the tools available to maintainers.

### PR DESCRIPTION
**Pull request by feanil**
Outline the tools maintainers have at their disposal and the purposes of
those tools.

### CODE REVIEW

File: source/developers/references/tools_for_maintainers.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Tools for Maintainers
+#####################
+
+.. list-table::
```
Comment (nedbat) [inline source/developers/references/tools_for_maintainers.rst:4]: We should include Discourse and GitHub issues in this list.  I think during the kick-off we were discussing channels for maintainers to monitor/broadcast.

File: source/developers/references/tools_for_maintainers.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+   * - `Slack Channel`_
+     - - Coordinating with other maintainers.
+       - Sharing experiences as a maintainer.
+   * - `Backstage`_
+     - - Seeing what you own.
+       - Seeing who ows what.
+       - Seeing how repos relate to each other.
+   * - `Grimoire`_
+     - - Monitor your personal PR Backlog.
+       - Moniter the PR backlog in repos you own.
```
Comment (mariajgrimaldi) [inline source/developers/references/tools_for_maintainers.rst:17]:
```suggestion
       - Monitor the PR backlog in repos you own.
```

File: source/developers/references/tools_for_maintainers.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+.. list-table::
+
+   * - Tool
+     - What to Use it For
+   * - `Slack Channel`_
+     - - Coordinating with other maintainers.
+       - Sharing experiences as a maintainer.
+   * - `Backstage`_
+     - - Seeing what you own.
+       - Seeing who ows what.
```
Comment (e0d) [inline source/developers/references/tools_for_maintainers.rst:13]: Spelling.

nit: maybe just "Seeing what other people own."

File: source/developers/references/tools_for_maintainers.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+       - Seeing how repos relate to each other.
+   * - `Grimoire`_
+     - - Monitor your personal PR Backlog.
+       - Monitor the PR backlog in repos you own.
+       - Make graphs of various github activity.
+   * - `Discourse`_
+     - - Discussing complex issues with the community.
```
Comment (e0d) [inline source/developers/references/tools_for_maintainers.rst:20]: Conversations that will be valuable to other community members in the future, like how to fix some non-unique problem?

**Review by e0d — APPROVED** [review]
One spelling fix, otherwise non-blocking feedback.

**Review by mariajgrimaldi — APPROVED** [review]

File: source/developers/references/tools_for_maintainers.rst [hunk 5]
```diff
@@ -0,0 +1,30 @@
+Tools for Maintainers
+#####################
+
+.. list-table::
+
+   * - Tool
+     - What to Use it For
+   * - `Slack Channel`_
+     - - Coordinating with other maintainers.
+       - Sharing experiences as a maintainer.
```
Comment (nedbat) [inline source/developers/references/tools_for_maintainers.rst:10]: Do we not expect users of components to be in Slack?

File: source/developers/references/tools_for_maintainers.rst [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+   * - `Backstage`_
+     - - Seeing what you own.
+       - Seeing who ows what.
+       - Seeing how repos relate to each other.
+   * - `Grimoire`_
+     - - Monitor your personal PR Backlog.
+       - Monitor the PR backlog in repos you own.
+       - Make graphs of various github activity.
```
Comment (nedbat) [inline source/developers/references/tools_for_maintainers.rst:18]:
```suggestion
       - Make graphs of various GitHub activity.
```

**Comment by feanil** [comment]
@nedbat added the rest of slack.

**Review by nedbat — APPROVED** [review]

**Pull request merged**